xds/src/main/java/io/grpc/xds/XdsClientImpl.java

```diff
@@ -86,7 +86,9 @@
   static final int INITIAL_RESOURCE_FETCH_TIMEOUT_SEC = 15;
 
   @VisibleForTesting
-  static final String ADS_TYPE_URL_LDS = "type.googleapis.com/envoy.api.v2.Listener";
+  static final String ADS_TYPE_URL_LDS_v2 = "type.googleapis.com/envoy.api.v2.Listener";
```

Comment:
Is the lowercase `v2` intended or typo? A const literal should be all uppercase, right? Same thing for other `_v2` literals below.

Reply:
Corrected.